app/assets/javascripts/channels/room.coffee

```diff
@@ -0,0 +1,13 @@
+App.room = App.cable.subscriptions.create "RoomChannel",
```

Comment:
Hi @namangupta01 this is great, and I have nothing against CoffeeScript but do you think we should try to standardize so there's not a mix of coffeescript/javascript in the project? I'm open to suggestions but just thinking about newcomers and how much they have to learn to be able to work on this file... what do you think?

Reply:
Sure! I will change it to javascript. 👍